spec/support/sqs.rb

```diff
@@ -1,24 +0,0 @@
-require 'aws-sdk'
-require 'fake_sqs/test_integration'
-
-# AWS Config for local fake SQS
-Aws.config.update(
-    region: "us-east-1",
-    credentials: Aws::Credentials.new("fake", "fake"),
-)
-
-db = ENV["SQS_DATABASE"] || ":memory:"
-puts "\n\e[34mRunning specs with database \e[33m#{db}\e[0m"
-
-$fake_sqs = FakeSQS::TestIntegration.new(
-    database: db,
-    sqs_endpoint: "localhost",
-    sqs_port: 4568,
-)
-
-RSpec.configure do |config|
-  # RSpec config for fake SQS
-  config.before(:each, :sqs) { $fake_sqs.start }
-  config.before(:each, :sqs) { $fake_sqs.reset }
-  config.after(:suite) { $fake_sqs.stop }
-end
```

Comment:
I know this got merged but actually I don't think we should have removed this. Now we don't have a local SQS queue and for running any tests with that queue we'd be testing against a third party service. That'll require genuine AWS credentials and a web connection. I solved that by having a local fake SQS that spun up only when tests had the :sqs metadata tag. I think that made perfect sense.